### PR TITLE
Update all dependencies to latest.

### PR DESCRIPTION
**Pull request by erossignon**

### CODE REVIEW

**Comment by robertsLando** [comment]
I would firstly move CI to github actions

**Comment by leerob** [comment]
Would love to! https://github.com/vercel/pkg/issues/1071

**Comment by leerob** [comment]
We've now added GitHub Actions for CI. Would you mind merging master?

**Comment by leerob** [comment]
Also - shall we close all the Dependabot PRs and instead manage all the updates here? We can use Dependabot going forward, but for the bulk update, let's do it all at once.

**Comment by jesec** [comment]
> Also - shall we close all the Dependabot PRs and instead manage all the updates here? We can use Dependabot going forward, but for the bulk update, let's do it all at once.

Totally agree. I don’t think we should spam the commit history with automated commits.

**Comment by robertsLando** [comment]
Once this is merged dependabot will automatically close other pr

**Comment by robertsLando** [comment]
@erossignon tests are failing

**Comment by robertsLando** [comment]
@erossignon Please fix conflicts and let's see if now it works

File: package.json [hunk 1]
```diff
@@ -59,26 +59,28 @@
     "babel-eslint": "10.1.0",
     "eslint": "7.22.0",
     "eslint-config-airbnb-base": "14.2.1",
+    "eslint-config-klopov": "^1.0.2",
```
Comment (leerob) [inline package.json:62]: We can remove this as it's not being used now, correct?

Comment (erossignon) [reply package.json:62]: correct

**Review by leerob — APPROVED** [review]
All tests pass! 🚀

**Pull request merged**